core/src/retransmit_stage.rs

```diff
@@ -252,6 +252,35 @@ fn enable_turbine_retransmit_peers_patch(shred_slot: Slot, root_bank: &Bank) ->
     }
 }
 
+// Drops shred slot leader from retransmit peers.
+// TODO: decide which bank should be used here.
+// TODO: confirm that this should use shred.slot() as opposed to
+// packet.meta.slot.
```

Comment:
They should be the same value. I think `packet.meta.slot` was an optimization to not deserialize the slot again out of the packet.

Reply:
thanks, removed the todo comment